packages/lab/src/avatar/Avatar.css

```diff
@@ -1,66 +1,36 @@
 /* Default variables applied to the root element */
 .saltAvatar {
-  --avatar-background: var(--saltAvatar-background, var(--salt-accent-background));
   --avatar-foreground: var(--saltAvatar-foreground, var(--salt-accent-foreground));
+  --avatar-size-multiplier: var(--saltAvatar-size-multiplier, 1);
+  --avatar-base-size: var(--salt-size-base, 12px);
+  --avatar-base-fontSize: var(--saltAvatar-fontSize, var(--salt-accent-fontSize));
+  --avatar-container-size: calc(var(--avatar-base-size) * var(--avatar-size-multiplier));
+  --avatar-fontSize: calc(var(--avatar-base-fontSize) * var(--avatar-size-multiplier));
+  /* Icon styling */
+  --saltIcon-color: var(--avatar-foreground);
+  --salt-size-icon-base: calc(var(--avatar-container-size) / 2);
 }
 
 /* Style applied to the root element */
 .saltAvatar {
```

Comment:
I think we should be adding line-height here (--salt-accent-lineHeight), i'm going to tag in figma specs also

Reply:
I tried `--salt-accent-lineHeight`, and it is not noticeable in the other densities, but it seems to be pushing the text up from the centre in high density, specially in size 1

Reply:
hmm, I'm not noticing a difference myself when I add it in storybook - could you screenshot?
I think even if it's not noticeable it's important in case it's nested inside other components that set the line height or the global line height is changed

Reply:
it was a tiny move, but I build again and it seems fine now, so i think it was something in my local :) 

![image](https://user-images.githubusercontent.com/13244486/222099766-268b3719-ea26-45dd-a825-06609a41ad4b.png)

lineHeight set: 
![image](https://user-images.githubusercontent.com/13244486/222099921-375de384-4f20-4598-8654-f3421739625b.png)